recipes/elastic-apm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "elastic-apm" %}
+{% set version = "6.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/elastic-apm-{{ version }}.tar.gz
+  sha256: 7e5669556d9a228d5508c3f4925e3a7efd0cbf9203a4655e287408aa6bc53a84
+
+build:
+  number: 0
+  noarch: python
+  script: ELASTIC_APM_WRAP_EXTENSIONS=false {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest
```

Comment:
Does this actually need pytest in order to build the package? It doesn't seem to be imported by `setup.py` (unless you run the pytest command)

Reply:
```suggestion
```